avivator/empty-fs.js

```diff
@@ -1,3 +1,5 @@
 export const close = '';
 export const open = '';
 export const read = '';
+const fs = '';
```

Comment:
Likewise. This empty-fs is for geotiff. It would be good to be very explicit about what modules we are resolving this way. Perhaps add a comment.